modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -365,17 +380,16 @@ sda.enkf.multisite <- function(settings,
         #for restart when t=1 do not need to do model runs and X should already exist in environment by this point
         X <- X
       }else{
-        
         if (control$debug) browser()
         #if restart then use restart.list, include site for debugging purposes
         if(restart_flag){
-          restart.arg = restart.list #$`646`debugging hack site specific
+          restart.arg = restart.list$`646`
```

Comment:
debugging hack reintroduced, needs to be undone

Reply:
Fixed.